### ISSUE TITLE
FetchLatestDSBlocks failed everytimes

### ISSUE THREAD

**Issue by cagiv** [Zilliqa/Zilliqa#1383]
Everytimes get this warning:
![warn_fetchlatestdsblocks](https://user-images.githubusercontent.com/47381377/52330268-561d5500-29f5-11e9-8ccd-146cce378b48.png)

**Comment by bb111189** [comment]
Have you enable UPnP or have port forwarded your router? Do refer to https://github.com/Zilliqa/Zilliqa/wiki/Mining#network-setup

**Comment by cagiv** [comment]
I do CPU-Mining on a VPS located in germany with Ubuntu 18.04, public IP-adress, 4 gb ram, 100 mbits/ports.  Do i have to make any settings here? Do you need more informations?
Thanks in advanced

**Comment by bmatthewshea** [comment]
Same problem.
Yes, my ports are open to public.
NOT a connectivity problem.

```
[WARN][  881][19-02-06T20:00:46.086][PoWProcessing.cpp:72][GetLatestDSBlock    ] FetchLatestDSBlocks Timeout... tried 1/3 times
[WARN][  881][19-02-06T20:01:31.086][PoWProcessing.cpp:72][GetLatestDSBlock    ] FetchLatestDSBlocks Timeout... tried 2/3 times
[WARN][  881][19-02-06T20:02:16.087][PoWProcessing.cpp:72][GetLatestDSBlock    ] FetchLatestDSBlocks Timeout... tried 3/3 times
[WARN][  881][19-02-06T20:02:16.087][PoWProcessing.cpp:80][GetLatestDSBlock    ] Fetch latest DS Block failed

```

**Comment by bb111189** [comment]
Was this issue present when you first launch the node or you occurs only after a couple of hours

**Comment by evesnow91** [comment]
@bmatthewshea That will indicate you are in sync, but not part of the validating network. You will need to mine to join the network every 100th TX block. Current difficulty level for shard is `31`, and current difficulty level for DS is `39`.

**Comment by cagiv** [comment]
@evesnow91 
>That will indicate you are in sync, but not part of the validating network. You will need to mine to join the network every 100th TX block.<
I also still constantly see `lowBlockNum is higher the highblocknum, maybe DS epoch ongoing`
Is it perhaps because the system / CPU is too weak / too slow for the difficulty?
Or it has nothing to do with it and it's just luck

**Comment by bb111189** [comment]
> @evesnow91
> You said: That will indicate you are in sync, but not part of the validating network. You will need to mine to join the network every 100th TX block.<<
> I also still constantly see `lowBlockNum is higher the highblocknum, maybe DS epoch ongoing`
> Is it perhaps because the system / CPU is too weak / too slow for the difficulty?
> Or it has nothing to do with it and it's just luck

You should be in sync. However, given the current difficulty of 31, CPU mining will not be effective in finding a PoW soln within the time window.

**Comment by bmatthewshea** [comment]
Bump
**Why am I am seeing 0 rewards? How often should I check?**
Please see my post above.

**Comment by bb111189** [comment]
> Thank you for info @evesnow91 👍 - Why are people telling us the port isn't open then? {sigh}
> 
> I have read mining doc a few times. Maybe I still don't understand? :
> Still not sure why I am receiving no rewards.
> I spin up about 80MH worth of miners when DS epoch hits. I submit PoW in log.
> I get this as already noted in other thread: `Failed to find soln for DS diff`
> BTW: My latency is generally good. My box is ntp time synced.
> 
> **Why am I am seeing 0 rewards? How often should I check?**
> 
> I also noted on the reward spreadsheet (docs link to it) shows you need 1.0+ GH to get 'promoted'!?
> 
> ![host22-2019-02-07_115035](https://user-images.githubusercontent.com/984097/52432023-0b700b80-2acf-11e9-83d3-3c9ef8237e69.png)


You may not have joined the network. 

To find out whether you have successfully entered the network, do refer to https://forum.zilliqa.com/t/my-node-is-running-normal-but-no-rewards-in-24-h-is-this-normal/363/7?u=junhao

**Comment by bmatthewshea** [comment]
cat zilliqa-00001-log.txt | grep -i "I am"
[INFO][ 1810][19-02-07T14:51:55.215][liqa/Zilliqa.cpp:276][operator()          ] I am a ds/normal node.

**Comment by bb111189** [comment]
Closing this issue as it is obselete
